app/controllers/FileChecksController.scala

```diff
@@ -119,16 +121,45 @@ class FileChecksController @Inject() (
         ).uncache()
       } else {
         if (isJudgmentUser) {
-          Ok(views.html.judgment.judgmentFileChecksProgress(consignmentId, reference, applicationConfig.frontEndInfo, request.token.name)).uncache()
+          if (applicationConfig.blockAutomateJudgmentTransfers) {
```

Comment:
Should ensure that the [feature block](https://github.com/nationalarchives/tdr-terraform-environments/pull/465) isn't merged or this may break.

Reply:
Yeah, I have removed this block